output/index.yaml

```diff
@@ -0,0 +1,110 @@
+openapi: 3.0.0
+info:
+  title: Sentences API
+  description: Sentences REST API
+  version: 0.1.1
+servers:
+- url: https://virtserver.swaggerhub.com/MHAMLIN/sentences/0.1.1
+  description: SwaggerHub API Auto Mocking
+paths:
+  /sentences:
+    get:
+      description: Returns a list of sentences
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Sentence'
+components:
+  schemas:
+    Sentence:
+      required:
+      - id
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the sentence
+          format: uuid
+        state:
+          pattern: "^\\d{2}"
+          type: string
+          description: State FIPS code
+        county:
+          pattern: "^\\d{5}"
+          type: string
+          description: County FIPS code
+        date:
+          type: string
+          description: "Sentencing date (month, day, and year)"
+          format: date
+        fine:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        restitution:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        confinement:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        suspension:
+          $ref: '#/components/schemas/SentenceType'
+        probation:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        community_service:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        treatment:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        death:
```

Comment:
what is death? death sentence? Not sure where this came from, but I think we may have lost life_sentence_indicator (not sure if that is what you meant or if this is a new type from CJARs - but this refers to whether the sentence is a life sentence).

Reply:
yeah. it was from CJARS. a death sentence type.